File: commons/test-utils/src/test/java/com/cedarsoft/test/utils/DisableWhenHeadlessConditionTest.java
package com.cedarsoft.test.utils;

import org.junit.jupiter.api.*;

/**
 */
class DisableWhenHeadlessConditionTest {
  @DisableIfHeadless
  @Test
  void testItIfHeadless() throws Exception {
    System.out.println("Running");
  }
}

File: commons/test-utils/src/test/kotlin/com/cedarsoft/test/utils/DisableWhenHeadlessConditionTest.kt
package com.cedarsoft.test.utils

import org.junit.jupiter.api.Test

/**
 */
internal class DisableWhenHeadlessConditionTest {
  @DisableIfHeadless
  @Test
  @Throws(Exception::class)
  fun testItIfHeadless() {
    println("Running")
  }
}
